lib/inspector-window.js

```diff
@@ -56,7 +60,7 @@ const InspectorWindow = Class(
     this.toolboxOverlay.removeTransportListener(this);
 
     if (this.win) {
-      this.win.destroy();
+      this.win.close();
```

Comment:
@janodvarko This fix a small bug on auto-closing and cleanup the rdp-inspector window when the developer toolbox is closed. (my apologies I forgot to split it in a separate commit)